packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,148 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+ /* eslint camelcase: [0] */
+
+import '../..';
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+const moment = require(`moment`);
+
+describe(`plugin-calendar`, () => {
+  describe(`#meetings()`, () => {
+
+    let createdMeeting, mccoy, organizer, spark, spock;
+
+    before(`create test users`, () => testUsers.create({
```

Comment:
search the various test suites to see how to call `spark.device.register()` in tests.